TurkishFilter/sections/specific.txt

```diff
@@ -428,6 +428,7 @@ koinmedya.com###custom_html-52
 ||i.ibb.co/2Pt8tqJ/turna.png
 ||i.ibb.co/0nCpbCv/otelz.jpg
 ||i.ibb.co/bXc58hB/turna.png
+||i.ibb.co/tJ9cJVx/turnaa.png
```

Comment:
Can you show this picture via opened devtools? The video haven't worked on my end.
You can also use this rule and remove similar ones.
```
||i.ibb.co/*/turna$image,domain=dizibox.pro
```

Reply:
Did you try that? https://www.dizibox.pro/maedeudok-1-sezon-4-bolum-izle/
or that. https://yabancidizi.pw/dizi/maedeudok/sezon-1/bolum-4
This AD is using on more than one website.